webpack.config.js

```diff
@@ -0,0 +1,66 @@
+const webpack = require("webpack");
+const path = require("path");
+const CopyWebpackPlugin = require("copy-webpack-plugin");
+const ExtractTextPlugin = require("extract-text-webpack-plugin");
+const package = require("./package");
+const widgetName = package.widgetName;
+
+const widgetConfig = {
+    entry: {
+        OfflineSearch: `./src/components/${widgetName}.ts`,
+    },
+    output: {
+        path: path.resolve(__dirname, "dist/tmp"),
+        filename: `src/com/mendix/widget/custom/${widgetName}/${widgetName}.js`,
```

Comment:
Warning! This will result in a wrong casing of the 
`src/com/mendix/widget/custom/OfflineSearch/OfflineSearch.js`
expect (as defined in the widget.xml)
`src/com/mendix/widget/custom/offlinesearch/OfflineSearch.js`